pkg/expectations/expectations.go

```diff
@@ -188,6 +188,11 @@ func (r *ControllerExpectations) SetExpectations(controllerKey string, add, del
 	return r.Add(exp)
 }
 
+// ExpectCreations wraps RaiseExpectations for the reciever controller specified, setting creates
```

Comment:
Don't think we even need this, I don't see it called anywhere.

Reply:
ah, oh yeah. This was left over from another idea i had. I'll remove